contributors/guide/README.md

```diff
@@ -234,6 +235,18 @@ When reviewing PRs from others [The Gentle Art of Patch Review](http://sage.thes
 
 Note: if your pull request isn't getting enough attention, you can use the [#pr-reviews](https://kubernetes.slack.com/messages/pr-reviews) channel on Slack to get help finding reviewers.
 
+## Contributions best practices
+
+- Do write clear and meaningful Git commit messages.
+- Make sure to include _Github Special Keywords_ which references the issue and automatically closes the issue when PR is merged.
```

Comment:
```suggestion
- If the PR will *completely* fix a specific issue, include `fixes #123` in the PR body (where 123 is the specific issue number the PR will fix. This will automatically close the issue when the PR is merged.
```